cfy_manager/components/restservice/scripts/create_tables_and_add_defaults.py

```diff
@@ -29,27 +28,24 @@
 from manager_rest.storage import db, models, get_storage_manager  # NOQA
 from manager_rest.amqp_manager import AMQPManager
 from manager_rest.flask_utils import setup_flask_app
-from manager_rest.storage.storage_utils import \
-    create_default_user_tenant_and_roles
+from manager_rest.storage import storage_utils
 
-logger = \
-    logging.getLogger('[{0}]'.format('create_tables_and_add_defaults'.upper()))
 CA_CERT_PATH = '/etc/cloudify/ssl/cloudify_internal_ca_cert.pem'
 
 
 def _init_db_tables(db_migrate_dir):
-    logger.info('Setting up a Flask app')
+    print('Setting up a Flask app')
```

Comment:
why?

Reply:
1. This is not expected by the script caller - it expects print calls.
2. The logger was never setup correctly here and didn't really output anything.